src/index.html

```diff
@@ -11,7 +11,194 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset>
+        <legend>Personal info:</legend>
+        <label class="form-field">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            required
+            minlength="5"
+            maxlength="25"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+            type="text"
+            name="name"
+            required
+            minlength="5"
+            maxlength="25"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            required
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="date-of-birth"
+            required
+          >
+        </label>
+
+        <div>
+          <label
+            class="form-label"
+            for="terms-and-conditions"
+          >
+            I accept the terms of the agreement
+          </label>
+          <input
+            id="terms-and-conditions"
+            type="checkbox"
+            name="terms_and_conditions"
+          >
+        </div>
+
+      </fieldset>
+
+      <fieldset>
+        <legend>
+          Registration
+        </legend>
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            required
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label>
+          Password:
+          <input
+            type="password"
+            name="password"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>
+          An interesting fact about you?
+        </legend>
+        <label class="form-field">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="cats"
+              value="y"
+            >
+            Yes
+          </label>
+
+          <label>
+            <input
+              type="radio"
+              name="cats"
+              value="n"
+            >
+            No
+          </label>
+
+        </label>
```

Comment:
Don't use this common label, because when you click on the question, yes becomes selected
```suggestion
        <div class="form-field">
          Do you love cats?
          <label>
            <input
              type="radio"
              name="cats"
              value="y"
            >
            Yes
          </label>

          <label>
            <input
              type="radio"
              name="cats"
              value="n"
            >
            No
          </label>
        </div>
```